creator-node/stage.env

```diff
@@ -24,8 +23,7 @@ logLevel=info
 maxAudioFileSizeBytes=1000000000
 pinAddCIDs=Qma5FGErNqHA32cyEjgp8hnwCFoqyYcBXU1ySKPA6MUJS5,QmYPy9A3zxLeBBuk1vpPNeG5aoMfa5qLNjJgh473Z9zV9b
 setTimeout=3600000
-snapbackHighestReconfigMode=RECONFIG_DISABLED
-snapbackMaxLastSuccessfulRunDelayMs=18000000 # 5hrs in ms
+stateMachineHighestReconfigMode=PRIMARY_AND_OR_SECONDARIES
```

Comment:
one note about this - this is actually as an override on stage nodes. we could clear it in one place. but also your PR description [here](https://github.com/AudiusProject/audius-protocol/pull/3716) is not entirely accurate but i don't think it'll cause any problems